memiavl/snapshot.go

```diff
@@ -122,18 +134,41 @@ func OpenSnapshot(snapshotDir string) (*Snapshot, error) {
 		)
 	}
 
+	// Read the plain offsets table from the end of keys file
+	keys := keysMap.Data()
+	offset := binary.LittleEndian.Uint64(keys[len(keys)-8:])
+	keysOffsets, err := NewPlainOffsetTable(keys[offset:])
+	if err != nil {
+		return nil, err
+	}
+
+	// Read the elias-fano offsets table from the end of values file
+	values := valuesMap.Data()
+	offset = binary.LittleEndian.Uint64(values[len(values)-8:])
+	valuesOffsets, _ := eliasfano32.ReadEliasFano(values[offset:])
+
+	nodesData, err := NewNodes(nodes)
```

Comment:
Now we only use little endian with build !nativebyteorder right?

Reply:
99% machines on the market has little endian as native endian, so they are almost equivalent practically, but `!nativebyteorder` is indeed more portable.

Reply:
oh right since I see no much diff when try `go test -run=^$ -bench=BenchmarkRandomGet -benchmem ./ -count=2 --tags=!`

Reply:
I think `!nativebyteorder` means when the tag is not set, you don't need to set `!` explicitly, when set `nativebyteorder` it's the native version, when not set, the portable little endian version.